content/includes/footer.html

```diff
@@ -140,7 +140,7 @@ <h3>Get help from Veterans Crisis Line</h3>
 (function() {
   var widgets = {{ widgets | json }};
   var module = {};
-  {% include "src/applications/static-pages/static-page-widgets.js" %}
+  {% include "assets/js/static-page-widgets.js" %}
   mountWidgets(widgets, '{{ buildtype }}' === 'production');
```

Comment:
I didn't want to have to move this, but it's a necessary step because after a full separation of the Metalsmith content from app-code, the Metalsmith build has to be able build content alone without accessing files within app-code.